### PR TITLE
Six Pack and ortho_ layout updates

### PR DESCRIPTION
**Pull request by Suschman**
Small changes to Six Pack, and improved ortho_2x3 support as i came along.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [X] Enhancement/optimization
- [X] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [X] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [X] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [X] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/sixkeyboard/rules.mk [hunk 1]
```diff
@@ -29,3 +29,4 @@ BLUETOOTH_ENABLE = no	# Enable Bluetooth with the Adafruit EZ-Key HID
 CUSTOM_MATRIX = yes
 
 SRC = matrix.c
```
Comment (fauxpark) [inline keyboards/sixkeyboard/rules.mk:31]:
```suggestion
SRC += matrix.c
```
Entirely unrelated, but may as well fix this :P

File: layouts/default/ortho_2x3/info.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "url": "",
+    "maintainer": "qmk",
+    "width": 3,
+    "height": 2,
+    "layouts": {
+        "LAYOUT_ortho_2x3": {
+          "layout": [
+                {"x":0, "y":0}, {"x":1, "y":0}, {"x":2, "y":0},
+                {"x":0, "y":1}, {"x":1, "y":1}, {"x":2, "y":1}
+          ]
```
Comment (fauxpark) [inline layouts/default/ortho_2x3/info.json:12]:
```suggestion
            "layout": [
                {"x":0, "y":0},
                {"x":1, "y":0},
                {"x":2, "y":0},

                {"x":0, "y":1},
                {"x":1, "y":1},
                {"x":2, "y":1}
            ]
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Comment by fauxpark** [comment]
Lint errors:

`☒ 40percentclub/5x5: Claims to support a community layout that does not exist: ortho_5x5`
Already removed in `develop`, can be ignored
`☒ 40percentclub/6lit: Claims to support a community layout that does not exist: ortho_2x6`
Needs to be added to `layouts/default`
`☒ Missing keyboards/handwired/stream_cheap/2x3/readme.md`
Readme exists in `handwired/stream_cheap`, can be ignored (the linter is not very smart here)

File: layouts/default/ortho_2x6/info.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    "url": "",
+    "maintainer": "qmk",
+    "width": 6,
+    "height": 2,
+    "layouts": {
+        "LAYOUT_ortho_2x6": {
+          "layout": [
+                {"x":0, "y":0}, {"x":1, "y":0}, {"x":2, "y":0}, {"x":3, "y":0}, {"x":4, "y":0}, {"x":5, "y":0},
+                {"x":0, "y":1}, {"x":1, "y":1}, {"x":2, "y":1}, {"x":3, "y":1}, {"x":4, "y":1}, {"x":5, "y":1}
+          ]
```
Comment (fauxpark) [inline layouts/default/ortho_2x6/info.json:12]:
```suggestion
            "layout": [
                {"x":0, "y":0},
                {"x":1, "y":0},
                {"x":2, "y":0},
                {"x":3, "y":0},
                {"x":4, "y":0},
                {"x":5, "y":0},

                {"x":0, "y":1},
                {"x":1, "y":1},
                {"x":2, "y":1},
                {"x":3, "y":1},
                {"x":4, "y":1},
                {"x":5, "y":1}
            ]
```

File: layouts/default/ortho_5x5/info.json [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+    "height": 5,
+    "layouts": {
+        "LAYOUT_ortho_5x5": {
+          "layout": [
+		{"x":0, "y":0}, {"x":1, "y":0}, {"x":2, "y":0}, {"x":3, "y":0}, {"x":4, "y":0},
+		{"x":0, "y":1}, {"x":1, "y":1}, {"x":2, "y":1}, {"x":3, "y":1}, {"x":4, "y":1},
+		{"x":0, "y":2},	{"x":1, "y":2}, {"x":2, "y":2}, {"x":3, "y":2}, {"x":4, "y":2},
+		{"x":0, "y":3}, {"x":1, "y":3}, {"x":2, "y":3}, {"x":3, "y":3}, {"x":4, "y":3},
+		{"x":0, "y":4}, {"x":1, "y":4}, {"x":2, "y":4}, {"x":3, "y":4}, {"x":4, "y":4}
+          ]
```
Comment (fauxpark) [inline layouts/default/ortho_5x5/info.json:15]:
```suggestion
            "layout": [
                {"x":0, "y":0},
                {"x":1, "y":0},
                {"x":2, "y":0},
                {"x":3, "y":0},
                {"x":4, "y":0},

                {"x":0, "y":1},
                {"x":1, "y":1},
                {"x":2, "y":1},
                {"x":3, "y":1},
                {"x":4, "y":1},

                {"x":0, "y":2},
                {"x":1, "y":2},
                {"x":2, "y":2},
                {"x":3, "y":2},
                {"x":4, "y":2},

                {"x":0, "y":3},
                {"x":1, "y":3},
                {"x":2, "y":3},
                {"x":3, "y":3},
                {"x":4, "y":3},

                {"x":0, "y":4},
                {"x":1, "y":4},
                {"x":2, "y":4},
                {"x":3, "y":4},
                {"x":4, "y":4}
            ]
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: layouts/default/ortho_5x5/info.json (hunk 4 above)
Comment (fauxpark) [inline layouts/default/ortho_5x5/info.json:15]:
```suggestion
            "layout": [
                {"x":0, "y":0},
                {"x":1, "y":0},
                {"x":2, "y":0},
                {"x":3, "y":0},
                {"x":4, "y":0},

                {"x":0, "y":1},
                {"x":1, "y":1},
                {"x":2, "y":1},
                {"x":3, "y":1},
                {"x":4, "y":1},

                {"x":0, "y":2},
                {"x":1, "y":2},
                {"x":2, "y":2},
                {"x":3, "y":2},
                {"x":4, "y":2},

                {"x":0, "y":3},
                {"x":1, "y":3},
                {"x":2, "y":3},
                {"x":3, "y":3},
                {"x":4, "y":3},

                {"x":0, "y":4},
                {"x":1, "y":4},
                {"x":2, "y":4},
                {"x":3, "y":4},
                {"x":4, "y":4}
            ]
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: layouts/default/ortho_5x5/info.json (hunk 4 above)
Comment (Suschman) [reply layouts/default/ortho_5x5/info.json]: Some day i will get this right at first try :/

File: layouts/default/ortho_5x5/info.json [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+		{"x":1, "y":3},
+		{"x":2, "y":3},
+		{"x":3, "y":3},
+		{"x":4, "y":3},
+
+		{"x":0, "y":4},
+		{"x":1, "y":4},
+		{"x":2, "y":4},
+		{"x":3, "y":4},
+		{"x":4, "y":4}
```
Comment (fauxpark) [inline layouts/default/ortho_5x5/info.json:38]:
```suggestion
            "layout": [
                {"x":0, "y":0},
                {"x":1, "y":0},
                {"x":2, "y":0},
                {"x":3, "y":0},
                {"x":4, "y":0},

                {"x":0, "y":1},
                {"x":1, "y":1},
                {"x":2, "y":1},
                {"x":3, "y":1},
                {"x":4, "y":1},

                {"x":0, "y":2},
                {"x":1, "y":2},
                {"x":2, "y":2},
                {"x":3, "y":2},
                {"x":4, "y":2},

                {"x":0, "y":3},
                {"x":1, "y":3},
                {"x":2, "y":3},
                {"x":3, "y":3},
                {"x":4, "y":3},

                {"x":0, "y":4},
                {"x":1, "y":4},
                {"x":2, "y":4},
                {"x":3, "y":4},
                {"x":4, "y":4}
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**